css/site.styl

```diff
@@ -378,8 +379,18 @@ img
   position: absolute
   top: 0
   left: 0
-  padding: $margin-small
-  z-index: 10
+  padding: $margin-tiny
+  margin-left: $margin-tiny
+  z-index: 100
+  color: $color-white
+  img 
```

Comment:
I think it would be better to give the image a class and target directly. That way you could also include the hover styles from line 358 and it would be more readable as well as performant